src/ImageSharp/MetaData/ImageMetaData.cs

```diff
@@ -99,6 +100,11 @@ public double VerticalResolution
             }
         }
 
+        /// <summary>
+        /// Gets or sets unit of measure used when reporting resolution.
+        /// </summary>
+        public PixelResolutionUnit ResolutionUnits { get; set; } = PixelResolutionUnit.PixelsPerInch;
```

Comment:
The docs of `HorizontalResolution` and `VerticalResolution` stills state *"[..] It is defined as the number of dots per inch [...]*. We should change this.

Reply:
I'll update it to `<see cref="PixelResolutionUnit.PixelsPerInch"/>` since that would be the default on a new instance.